src/hooks/useDitto.ts

```diff
@@ -62,13 +64,13 @@ export const useDitto = (props: useDittoProps) => {
       `Frame "${frameId}" not found this project "${projectId}"`
     );
 
-  if (!blockId) return filterFrame(frame, filters);
+  if (!blockId) return filterFrame(frame, variables, count, filters);
 
   const block = frame.blocks[blockId];
   if (!block)
     return nullError(
       `Block "${blockId}" not found in frame "${frameId}" in project "${projectId}"`
     );
 
-  return filterBlock(block, filters);
+  return filterBlock(block, variables, count, filters);
```

Comment:
```suggestion
  return filterBlock(block, variables || {}, count, filters);
```
For reasons described [here](https://github.com/dittowords/ditto-react/pull/10/files#r784865890)

Reply:
👍